javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +92,1316 @@ private API() {
+     * @param result The result.
+     * @return A succeeded {@code Future}.
+     */
+    public static <T> Future<T> Future(T result) {
+        return Future.successful(result);
+    }
+
+    /**
+     * Alias for {@link Future#successful(ExecutorService, Object)}
+     *
+     * @param <T>             The value type of a successful result.
+     * @param executorService An {@code ExecutorService}.
+     * @param result          The result.
+     * @return A succeeded {@code Future}.
+     * @throws NullPointerException if executorService is null
+     */
+    public static <T> Future<T> Future(ExecutorService executorService, T result) {
+        return Future.successful(executorService, result);
+    }
+
+    /**
+     * Alias for {@link Future#failed(Throwable)}
+     *
+     * @param <T>       The value type of a successful result.
+     * @param exception The reason why it failed.
+     * @return A failed {@code Future}.
+     * @throws NullPointerException if exception is null
+     */
+    public static <T> Future<T> Future(Throwable exception) {
+        return Future.failed(exception);
+    }
+
+    /**
+     * Alias for {@link Future#failed(ExecutorService, Throwable)}
+     *
+     * @param <T>             The value type of a successful result.
+     * @param executorService An executor service.
+     * @param exception       The reason why it failed.
+     * @return A failed {@code Future}.
+     * @throws NullPointerException if executorService or exception is null
+     */
+    public static <T> Future<T> Future(ExecutorService executorService, Throwable exception) {
+        return Future.failed(executorService, exception);
+    }
+
+    /**
+     * Alias for {@link Lazy#of(Supplier)}
+     *
+     * @param <T>      type of the lazy value
+     * @param supplier A supplier
+     * @return A new instance of Lazy
+     */
+    public static <T> Lazy<T> Lazy(Supplier<? extends T> supplier) {
+        return Lazy.of(supplier);
+    }
+
+    /**
+     * Alias for {@link Option#of(Object)}
+     *
+     * @param <T>   type of the value
+     * @param value A value
+     * @return {@code Some(value)} if value is not {@code null}, {@code None} otherwise
+     */
+    public static <T> Option<T> Option(T value) {
+        return Option.of(value);
+    }
+
+    /**
+     * Alias for {@link Option#some(Object)}
+     *
+     * @param <T>   type of the value
+     * @param value A value
+     * @return {@code Some(value)}
+     */
+    public static <T> Option<T> Some(T value) {
+        return Option.some(value);
+    }
+
+    /**
+     * Alias for {@link Option#none()}
+     *
+     * @param <T> component type
+     * @return the single instance of {@code None}
+     */
+    public static <T> Option<T> None() {
+        return Option.none();
+    }
+
+    /**
+     * Alias for {@link Option#nothing()}
+     *
+     * @return {@code Some<Void>}
+     */
+    public static Option<Void> Nothing() {
```

Comment:
Yes I remember discussion. But in that time we decide to include this method.
Will be removed